devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusPlatformTask.java

```diff
@@ -66,12 +65,11 @@ protected QuarkusPlatformDescriptor platformDescriptor() {
     }
 
     @Internal
-    protected GradleBuildFile getGradleBuildFile() {
-        final ProjectWriter writer = new FileProjectWriter(getProject().getProjectDir());
+    protected BuildFile getGradleBuildFile() {
         return getProject().getParent() == null
-                ? new GradleBuildFile(writer)
-                : new GradleBuildFile(writer,
-                        new FileProjectWriter(getProject().getRootProject().getProjectDir()));
+                ? new GradleBuildFileFromConnector(getProject().getProjectDir().toPath(), platformDescriptor())
```

Comment:
this code could be more readable to put `getProject().getRootProject().getProjectDir()`  in var.

Reply:
Yeah that's better indeed!